geode-core/src/main/java/org/apache/geode/internal/cache/wan/serial/SerialGatewaySenderEventProcessor.java

```diff
@@ -638,6 +639,17 @@ protected boolean basicHandlePrimaryDestroy(final EventID eventId) {
         ew.event.release();
         statistics.incUnprocessedEventsRemovedByPrimary();
         return true;
+      } else if (addToUnprocessedTokens) {
+        // Secondary event may not have arrived
+        Long mapValue =
+            Long.valueOf(System.currentTimeMillis() + AbstractGatewaySender.TOKEN_TIMEOUT);
```

Comment:
This can just be `Long mapValue = System.currentTimeMillis() + AbstractGatewaySender.TOKEN_TIMEOUT;`